java-surefire/src/main/java/org/sonar/plugins/surefire/data/SurefireStaxHandler.java

```diff
@@ -87,7 +87,8 @@ private static UnitTestResult parseTestResult(SMInputCursor testCaseCursor) thro
     detail.setName(name);
 
     String status = UnitTestResult.STATUS_OK;
-    long duration = getTimeAttributeInMS(testCaseCursor);
+    String time = testCaseCursor.getAttrValue("time");
```

Comment:
@m-g-sonar it needs to be defined before, `testCaseCursor` moves forward in ` testCaseCursor.descendantElementCursor()` and won't be able to return time attribute later.